### PR TITLE
TaxRate on the Product model should be a choice field

### PR DESCRIPTION
**Pull request by axce1**
I want to merge this change because...

TaxRate on the Product model should be a choice field #2829

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.

### CODE REVIEW

File: saleor/product/migrations/0072_auto_20180929_0707.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.8 on 2018-09-29 12:07
```
Comment (Pacu2) [inline saleor/product/migrations/0072_auto_20180929_0707.py]: Could you please rebase with the latest master? I think we are already on 72nd migration.

File: tests/api/test_product.py [hunk 2]
```diff
@@ -419,7 +419,7 @@ def test_create_product(
     assert data['product']['description'] == product_description
     assert data['product']['isPublished'] == product_isPublished
     assert data['product']['chargeTaxes'] == product_chargeTaxes
-    assert data['product']['taxRate'] == product_taxRate.lower()
+    assert data['product']['taxRate'] == product_taxRate
```
Comment (Pacu2) [inline tests/api/test_product.py]: I think we should use `TaxRateType.STANDARD.name` (graphql/core/types/money.py) instead of passing hardcoded string

Comment (axce1) [reply tests/api/test_product.py]: D'oh!!!
I will fix it

**Comment by maarcingebala** [comment]
Hey @axce1, your PR contains changes that were already merged to master. Could you pull the latest master and rebase your branch against it? You can only pick changes that you've made using `git rebase --interactive master`. Also, there is a conflict in one file.

**Comment by Pacu2** [comment]
Looks good to me, @axce1 could you please rebase against the latest master?

**Comment by axce1** [comment]
@Pacu2 I tried, but did not succeed
may i create new PR?

**Comment by Pacu2** [comment]
@axce1 I don't think that creating a new PR is necessary. 
Assuming that your local branch is in the same state as this one, just do
`git pull origin master --rebase`
Assuming that you have `https://github.com/mirumee/saleor.git` added as a `origin`

**Comment by axce1** [comment]
thx, I think I did it

why so many commits? may be something wrong?

**Comment by Pacu2** [comment]
Right, there should be only commits containing changes you've introduced. 

Not sure what you did wrong, but you might want to:
- note down hashes of commits you've introduced
- reset hard to mirumee/master
- cherry-pick the commits you've noted at first

Here's a nice git tutorial you might want to check:
https://try.github.io/

**Comment by Pacu2** [comment]
@axce1 please let me know if you're able to finish this PR, or shall we

**Comment by axce1** [comment]
@Pacu2 sorry, i try to finish next week

**Comment by Pacu2** [comment]
Closing for now. Please reopen if you'll have time to finish this PR!

**Pull request closed without merging**